### PR TITLE
Support for passing in a docker image for the frontend server

### PR DESCRIPTION
**Pull request by zeroasterisk**
https://github.com/kadirahq/meteor-up/issues/284

> I want to tweak my nginx config... the simplest thing for you would be to allow me to make my own image and change it.

### CODE REVIEW

File: src/modules/meteor/assets/templates/start.sh [hunk 1]
```diff
@@ -45,5 +45,5 @@ docker run \
     --link=$APPNAME:backend \
     --publish=<%= sslConfig.port %>:443 \
     --name=$APPNAME-frontend \
-    meteorhacks/mup-frontend-server /start.sh
+    <%= docker.imageFrontendServer %> /start.sh
```
Comment (madushan1000) [inline src/modules/meteor/assets/templates/start.sh:48]: Can you also fix the docker pull line above?

Comment (zeroasterisk) [reply src/modules/meteor/assets/templates/start.sh:48]: done

File: src/modules/meteor/assets/templates/start.sh [hunk 2]
```diff
@@ -15,6 +15,7 @@ docker rm -f $APPNAME-frontend
 # We don't need to fail the deployment because of a docker hub downtime
 set +e
 docker pull <%= docker.image %>
+docker pull <%= docker.imageFrontendServer %>
```
Comment (madushan1000) [inline src/modules/meteor/assets/templates/start.sh]: Not here, take a look at line 38/39

Comment (zeroasterisk) [reply src/modules/meteor/assets/templates/start.sh]: ok - sorry about that, updated

**Review by madushan1000 — APPROVED** [review]

**Pull request merged**